src/Nethermind/Nethermind.Blockchain/Data/EmptyLocalDataSource.cs

```diff
@@ -21,7 +21,7 @@ namespace Nethermind.Blockchain.Data
 {
     public class EmptyLocalDataSource<T> : ILocalDataSource<T>
     {
-        public T Data { get; } = default;
```

Comment:
`= default` is implied here, so having it is redundant.  Is it a stylistic choice to include it, or just an oversight?

Reply:
Probably either the author was thinking about something else or the code looked different before. It is all shown in Rider when we edit and is a matter of when it will be cleaned.